modules/lib/global.d.ts

```diff
@@ -1,210 +1,112 @@
-interface XpLibraries {}
+import type {
+    App,
+    DoubleUnderscore,
+    Log,
+    NewBean as NewBeanType,
+    Resolve,
+    ScriptValue as ScriptValueInterface,
+    XpRequire,
+} from '@enonic-types/core';
+
+
+//──────────────────────────────────────────────────────────────────────────────
+// Declare global types
+//──────────────────────────────────────────────────────────────────────────────
+declare global {
+    // TODO: Remove these two? They are now exported by @enonic-types/core and
```

Comment:
Yes, I think we should remove them, but it will be potential breaking change.

Reply:
I believe we decided a while ago the "breaking changes" in the type system, was ok.

So I'm removing them.